dashboard/app/views/layouts/_header.html.haml

```diff
@@ -25,6 +25,18 @@
               = render partial: 'shared/header_progress'
             - elsif @edit_blocks
               .header_level{style:'font-size: 250%; line-height: 125%;'} Editing #{@edit_blocks.humanize}
-          - if params[:controller] !~ /^sessions/
-            .header_user
-              = render partial: 'shared/user_header'
+          .header_right
+            %span.stage_links{style: 'display: none;'}
```

Comment:
They're `display: none;` by default, then they get shown based on the `scriptData` JSON here: https://github.com/code-dot-org/code-dot-org/pull/1154/files#diff-3bcf40f621bb1e4fc4cb234be362c2a9L40